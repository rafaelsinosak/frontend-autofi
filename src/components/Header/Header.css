.root {
  flex-grow: 1;
}
.menuButton {
  margin-right: themespacing(2);
}

.title {
  flex-grow: 1;
}
