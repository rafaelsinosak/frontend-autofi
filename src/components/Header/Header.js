import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";
import "./Header.css";

const Header = () => {
  return (
    <div className="root">
      <AppBar position="static">
        <Toolbar>
          <Typography variant="h6" className="title">
            AutoFi
          </Typography>
        </Toolbar>
      </AppBar>
    </div>
  );
};

export default Header;
