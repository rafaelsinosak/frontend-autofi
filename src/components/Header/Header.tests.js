import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

test('Header - Render Component', () => {
  const component = renderer.create(
    <Header></Header>,
  );
  let header = component.toJSON();
  expect(header).toMatchSnapshot();
});