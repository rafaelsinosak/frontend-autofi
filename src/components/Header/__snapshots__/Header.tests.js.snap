// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header - Render Component 1`] = `
<div
  className="root"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h6
        className="MuiTypography-root title MuiTypography-h6"
      >
        AutoFi
      </h6>
    </div>
  </header>
</div>
`;
